bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-messages/component.jsx

```diff
@@ -83,29 +83,95 @@ const intlMessages = defineMessages({
 });
 
 class UserMessages extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      index: -1,
+    };
+
+    this.openChatRefs = [];
+    this.selectedIndex = -1;
+
+    this.focusOpenChatItem = this.focusOpenChatItem.bind(this);
+    this.changeState = this.changeState.bind(this);
+  }
+
   componentDidMount() {
     if (!this.props.compact) {
       this._msgsList.addEventListener(
         'keydown',
-        event => this.props.rovingIndex(
+        event => this.props.roving(
           event,
-          this._msgsList,
-          this._msgItems,
           this.props.openChats.length,
+          this.changeState,
         ),
       );
     }
   }
 
-  render() {
+  componentDidUpdate(prevProps, prevState) {
+    if (this.state.index === -1) {
+      return;
+    }
+
+    if (this.state.index !== prevState.index) {
+      this.focusOpenChatItem(this.state.index);
+    }
+  }
+
+  getOpenChats() {
     const {
       openChats,
       openChat,
-      intl,
       compact,
       isPublicChat,
     } = this.props;
 
+    let index = -1;
+
+    return openChats.map(chat => (
+      <CSSTransition
```

Comment:
<CSSTransition> is not going to work on its own, it has to be wrapped into <TransitionGroup>, then it will be animated automatically.. you forgot this step.